### DISCUSSION TITLE
Change default Times New Roman font to another system font

### DISCUSSION THREAD

**Discussion by grelas** [fabricjs/fabric.js#7245] [Q&A]
Is there a way to override the default "Times New Roman" font for all objects to be a different system font - e.g. "Arial"?

**Comment by i-prikot** [comment]
Hi.

This should help you `fabric.Text.prototype.fontFamily = 'Arial'`

**Reply by asturur** [reply]
Yes this is a way and is correct.
@grelas was this what you were looking for?

**Reply by grelas** [reply]
Thanks! This ended up not being what I needed to do, but great to know!
